deps/ncrypto/ncrypto.cc

```diff
@@ -3813,4 +3856,93 @@ DataPointer hashDigest(const Buffer<const unsigned char>& buf,
   return data.resize(result_size);
 }
 
+// ============================================================================
+
+X509Name::X509Name() : name_(nullptr), total_(0) {}
```

Comment:
Nit suggestion:

Why don't you give default values to these attributes to have a default constructor and remove this line?